filter/series_by_tag.go

```diff
@@ -179,14 +214,14 @@ func createMatchingHandlerForOneTag(spec TagSpec) MatchingHandler {
 		}
 	}
 
-	matchEmpty := matchingHandlerCondition("")
+	// matchEmpty := matchingHandlerCondition("")
```

Comment:
is it ok?

Reply:
I want to remove it, but I need @msaf1980 to check graphite-clickhouse and give a review about some errors that might depend on this.